lang/funcs/funcgen/func.go

```diff
@@ -28,42 +28,34 @@ import (
 )
 
 type function struct {
-	MgmtPackage string     `yaml:"mgmtPackage"`
-	MgmtName    string     `yaml:"mgmtName"`
-	Help        string     `yaml:"help"`
-	GoPackage   string     `yaml:"goPackage"`
-	GoFunc      string     `yaml:"goFunc"`
-	Args        []arg      `yaml:"args"`
-	Return      []arg      `yaml:"return"`
-	Tests       []functest `yaml:"tests"`
+	MgmtPackage  string    `yaml:"mgmtPackage"`
```

Comment:
docstrings please so we know how to use this in the future.

Reply:
done